.circleci/config.yml

```diff
@@ -26,15 +26,15 @@ jobs:
           path: ./coverage
   integration-tests:
     docker:
-      - image: circleci/node:10
+      - image: circleci/node:12
```

Comment:
Please also update `Dockerfile` node version to node:12

Reply:
put it back to 10
will update another time.
don't want to run to issues now